packages/slate-theme/src/scripts/slate/rte.js

```diff
@@ -15,7 +15,9 @@ slate.rte = {
    * @param {string} options.tableWrapperClass - table wrapper class name
    */
   wrapTable: function(options) {
-    options.$tables.wrap('<div class="' + options.tableWrapperClass + '"></div>');
+    var tableWrapperClass = typeof options.tableWrapperClass !== 'string' ? '' : options.tableWrapperClass;
```

Comment:
If some passes `1234` then would result in a empty string, not `'1234'`. Is this okay? I'm on the fence.

Reply:
That's a fair concern.  I can change it back to checking `typeof undefined` and then outputting the value always as a string.

Regarding the integer class names: Having class names that _starts_ with a number requires some work arounds to function in a stylesheet - [quick codepen here](https://codepen.io/NathanPJF/pen/rGjNxg).  But those workarounds exist, so I shouldn't discount it.